### PR TITLE
Example functions clean up

### PR DESCRIPTION
**Pull request by PavelK27**
Some PHPCS fixes for example-functions file

### CODE REVIEW

File: example-functions.php [hunk 1]
```diff
@@ -24,13 +24,13 @@
...
- * @param  CMB2 object $cmb CMB2 object
+ * @param  CMB2 object $cmb CMB2 object.
  *
  * @return bool             True if metabox should show
  */
 function yourprefix_show_if_front_page( $cmb ) {
-	// Don't show this metabox if it's not the front page template
-	if ( $cmb->object_id !== get_option( 'page_on_front' ) ) {
+	// Don't show this metabox if it's not the front page template.
+	if ( get_option( 'page_on_front' !== $cmb->object_id ) ) {
```
Comment (jtsternberg) [inline example-functions.php]: This is not correct (and is broken)

File: example-functions.php [hunk 2]
```diff
@@ -65,10 +65,10 @@ function yourprefix_render_row_cb( $field_args, $field ) {
 	$value       = $field->escaped_value();
 	$description = $field->args( 'description' );
 	?>
-	<div class="custom-field-row <?php echo $classes; ?>">
-		<p><label for="<?php echo $id; ?>"><?php echo $label; ?></label></p>
-		<p><input id="<?php echo $id; ?>" type="text" name="<?php echo $name; ?>" value="<?php echo $value; ?>"/></p>
-		<p class="description"><?php echo $description; ?></p>
+	<div class="custom-field-row <?php echo esc_attr( $classes ); ?>">
+		<p><label for="<?php echo esc_attr( $id ); ?>"><?php echo esc_html( $label ); ?></label></p>
+		<p><input id="<?php echo esc_attr( $id ); ?>" type="text" name="<?php echo esc_attr( $name ); ?>" value="<?php echo esc_attr( $value ); ?>"/></p>
```
Comment (jtsternberg) [inline example-functions.php]: $value is already escaped, as it's taken from `$field->escaped_value()`

File: example-functions.php [hunk 3]
```diff
@@ -77,22 +77,22 @@ function yourprefix_render_row_cb( $field_args, $field ) {
...
- * @param  CMB2_Field $field      The field object
+ * @param  CMB2_Field $field      The field object.
  */
 function yourprefix_display_text_small_column( $field_args, $field ) {
 	?>
-	<div class="custom-column-display <?php echo $field->row_classes(); ?>">
-		<p><?php echo $field->escaped_value(); ?></p>
-		<p class="description"><?php echo $field->args( 'description' ); ?></p>
+	<div class="custom-column-display <?php echo esc_attr( $field->row_classes() ); ?>">
+		<p><?php echo esc_html( $field->escaped_value() ); ?></p>
```
Comment (jtsternberg) [inline example-functions.php]: `esc_html ` is not needed on a  call to `$field->escaped_value()`

**Comment by PavelK27** [comment]
The update was corrected

**Comment by jtsternberg** [comment]
It works, thanks!

**Pull request merged**